requirements/usage/usage-scenarios.tex

```diff
@@ -0,0 +1,26 @@
+\subsubsection{Usage scenarios}
+\begin{enumerate}
+  \item High priority task.
+  \begin{itemize}
+    \item Bob has to do some urgent work.
+    \item Bob inputs the task and data. He also sets the priority option to high.
+    \item System checks if the task can be directly executed somewhere.
+    If that is the case, the task is executed and the scenario ends.
+    \item System checks if enough resources can be freed for Bob's task by pausing one or more appropriate running tasks.
+    \item System ensures that the paused tasks will be continued when that is possible.
+    \item If there are no appropriate tasks to pause, the User is notified that he has to wait.
+  \end{itemize}
+  \item User checks the state of the task.
+  \begin{itemize}
+    \item Bob schedules a task that he has not thoroughly tested.
+    Bob is concerned that the task will not work as intended.
+    \item System has available resources, so it starts the task.
+    \item After 15 minutes Bob wants to check how the task is going.

```

Comment:
This sentence sounds strange.
I suggest the following wording:

After 15 minutes Bob wants to check if the task is working as intended.